WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ShippingLabelCustomsFormList.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelCustomsFormList: View {
+    @ObservedObject private var viewModel: ShippingLabelCustomsFormListViewModel
```

Comment:
It makes a lot of sense to me 👍 I would have used the same approach to manage the customs list